src/content/configuration/resolve.mdx

```diff
@@ -716,7 +716,7 @@ module.exports = {
 
 ### resolve.unsafeCache
 
-`RegExp` `[RegExp]` `boolean: true`
+`object` `boolean: true`
```

Comment:
We use `=` to indicate [the default value](https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types).
```suggestion
`object` `boolean = true`
```